docs/tutorial/Similarity Scores.ipynb

```diff
@@ -812,16 +818,16 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "[[0, 76], [1, 2345], 1.0]\n",
-      "[[0, 83], [1, 3439], 1.0]\n",
-      "[[0, 103], [1, 863], 1.0]\n",
-      "[[0, 154], [1, 2391], 1.0]\n",
-      "[[0, 177], [1, 4247], 1.0]\n",
-      "[[0, 192], [1, 1176], 1.0]\n",
-      "[[0, 270], [1, 4516], 1.0]\n",
-      "[[0, 312], [1, 1253], 1.0]\n",
-      "[[0, 407], [1, 3743], 1.0]\n",
-      "[[0, 670], [1, 3550], 1.0]\n"
+      "[76, 2345, 1.0]\n",
```

Comment:
Now we arrive at the problem with tutorials not compatible with the current branch: what is returned by the currently deployed service at `testing.es.data61.xyz` is not compatible with the version on this branch where the similarity score output type has changed. So to test your tutorial for the state of this branch, you have to build and deploy the entity service locally and test against (actually, no need to build as the docker images should already be on dockerhub for this branch).